.github/workflows/dbt-checks.yaml

```
@@ -0,0 +1,57 @@
+
+      - name: Setup Python version
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.9"
+
+      - name: Install requirements
+        run: |-
+          pip install --no-cache-dir -r requirements.txt
+
+      - name: Install sqlfmt
+        run: pip install shandy-sqlfmt[jinjafmt]
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2102:info:1:26: Ranges can only match single chars (mentioned due to duplicates)

(shellcheck)